ocrd_tesserocr/recognize.py

```diff
@@ -357,35 +383,147 @@ def _process_glyphs_in_word(self, result_it, word, word_xywh):
                 glyph_no += 1
                 result_it.Next(RIL.SYMBOL)
 
+def page_element_unicode0(element):
+    if element.get_TextEquiv():
+        return element.get_TextEquiv()[0].Unicode
+    else:
+        return ''
+
+def page_element_conf0(element):
+    if element.get_TextEquiv():
+        # generateDS does not convert simpleType for attributes (yet?)
+        return float(element.get_TextEquiv()[0].conf or "1.0")
+    return 1.0
+
+def page_get_reading_order(ro, rogroup):
+    '''Add all elements from the given reading order group to the given dictionary.
```

Comment:
Double quotes are used elsewhere.

Reply:
For the docstring? IIUC single and double quotes are exactly the same for triple-quote-delimited strings.

Reply:
So, do you want me to change it, or resolved?

Reply:
Despite the semantics, I think we should consistent at least within single files. So pls. change.

Reply:
cf 
66da5f6